Neos.ContentRepository.Export/src/Processors/EventStoreImportProcessor.php

```diff
@@ -106,74 +92,17 @@ public function run(): ProcessorResult
                 )
             );
             if (in_array($domainEvent::class, [ContentStreamWasCreated::class, ContentStreamWasForked::class, ContentStreamWasRemoved::class], true)) {
-                return ProcessorResult::error(sprintf('Failed to read events. %s is not expected in imported event stream.', $event->type));
+                throw new \RuntimeException(sprintf('Failed to read events. %s is not expected in imported event stream.', $event->type), 1729506757);
             }
             $domainEvent = DecoratedEvent::create($domainEvent, eventId: EventId::fromString($event->identifier), metadata: $event->metadata);
             $domainEvents[] = $this->eventNormalizer->normalize($domainEvent);
         }
 
-        assert($this->contentStreamId !== null);
-
-        $contentStreamStreamName = ContentStreamEventStreamName::fromContentStreamId($this->contentStreamId)->getEventStreamName();
-        $events = Events::with(
-            $this->eventNormalizer->normalize(
-                new ContentStreamWasCreated(
-                    $this->contentStreamId,
-                )
-            )
-        );
+        $contentStreamStreamName = ContentStreamEventStreamName::fromContentStreamId($workspace->currentContentStreamId)->getEventStreamName();
         try {
-            $contentStreamCreationCommitResult = $this->eventStore->commit($contentStreamStreamName, $events, ExpectedVersion::NO_STREAM());
+            $this->eventStore->commit($contentStreamStreamName, Events::fromArray($domainEvents), ExpectedVersion::ANY());
```

Comment:
`ExpectedVersion::ANY()` is not desired here, we should change that to
```suggestion
            $this->eventStore->commit($contentStreamStreamName, Events::fromArray($domainEvents), ExpectedVersion::fromVersion(Version::first()));
```